velox/dwio/parquet/reader/PageReader.h

```diff
@@ -489,6 +496,7 @@ class PageReader {
   std::unique_ptr<StringDecoder> stringDecoder_;
   std::unique_ptr<BooleanDecoder> booleanDecoder_;
   std::unique_ptr<DeltaBpDecoder> deltaBpDecoder_;
+  std::unique_ptr<TimestampDecoder> timestampDecoder_;
```

Comment:
I would not put this at decoder level, this should be a normal int64 decoder and you should convert it into `Timestamp` in column reader

Reply:
I have removed the TimestampDecoder class. And moved the logic to direct decoder.